static/docs/get-started/agenda.md

```diff
@@ -1,36 +1,35 @@
 # Agenda
 
-In the next few sections we will build a simple natural language processing
-(NLP) project from scratch. If you'd like to get the final result or have some
-issues along the way, you can download the fully reproducible
-[GitHub project](https://github.com/iterative/example-get-started) with:
+In the next few sections we'll build a simple natural language processing (NLP)
+project from scratch. If you'd like to get the final result or have any issues
+along the way, you can download the fully reproducible
+[GitHub project](https://github.com/iterative/example-get-started) by running:
 
 ```dvc
 $ git clone https://github.com/iterative/example-get-started
 ```
 
-Otherwise, bear with us and we will introduce the basic DVC concepts and get to
+Otherwise, bear with us and we'll introduce some basic DVC concepts to get to
 the same result together!
 
-The idea of the project is a simplified version of the
-[Tutorial](/doc/tutorials/deep). It explores the NLP problem of predicting tags
-for a given StackOverflow question. For example, we want a classifier that can
-predict posts about the Python language by tagging them `python`.
+The idea of the project is a simplified version of our
+[text classification tutorial](/doc/tutorials/deep). It explores the NLP problem
```

Comment:
Thanks for the update. Let's call this "Deep Dive Tutorial" please, as in the navigation.